src/scss/_colors.scss

```diff
@@ -0,0 +1,88 @@
+// PatternFly Colors
```

Comment:
Good call on this especially if we wind up tweaking colors to look better on bluejeans/projectors